### PR TITLE
Symfony 4.4/5.0 compatibility

### PR DESCRIPTION
**Pull request by Deuchnord**
#hackdayParis

### CODE REVIEW

**Comment by Deuchnord** [comment]
Having a weird issue with the event dispatcher, both on v4.4 and v5.0, when running the tests:
```
Lexik\Bundle\JWTAuthenticationBundle\Tests\DependencyInjection\LexikJWTAuthenticationExtensionTest::testEncoderConfiguration
Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: The service "lexik_jwt_authentication.security.authentication.provider" has a dependency on a non-existent service "event_dispatcher".
```

Tried to add `symfony/event-dispatcher` in the dev dependencies, but didn’t fix it.

Plus, I have two failing tests on my laptop, which seem to pass on the CI, and I don’t get why.

```1) Lexik\Bundle\JWTAuthenticationBundle\Tests\Services\KeyLoader\OpenSSLKeyLoaderTest::testLoadInvalidPublicKey
Failed asserting that exception message 'Failed to load public key: 
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line
 0909006C:PEM routines:get_name:no start line' contains '0906D06C:PEM routines:PEM_read_bio:no start line'.

2) Lexik\Bundle\JWTAuthenticationBundle\Tests\Services\KeyLoader\OpenSSLKeyLoaderTest::testLoadInvalidPrivateKey
Failed asserting that exception message 'Failed to load private key: 
 0909006C:PEM routines:get_name:no start line' contains '0906D06C:PEM routines:PEM_read_bio:no start line'.
```

Is it possible that’s related to my version of OpenSSL (OpenSSL 1.1.1d  10 Sep 2019)?

@chalasr WDYT?

File: composer.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "keywords": ["Symfony", "bundle", "jwt", "jws", "authentication", "api", "rest"],
     "homepage": "https://github.com/lexik/LexikJWTAuthenticationBundle",
     "license": "MIT",
+    "minimum-stability": "dev",
```
Comment (chindit) [inline composer.json]: Not sure it's a good idea to change this.

Comment (Deuchnord) [reply composer.json]: It's only to allow me working with Symfony 4.4 and Symfony 5 that have not been released yet. Once done, I'll remove it ;)

**Comment by sergiu-popa** [comment]
How did you install it for Symfony 5?
```
Problem 1
- lexik/jwt-authentication-bundle v2.6.4 requires symfony/framework-bundle ^3.4|^4.0 -> no matching package found.
```

**Comment by chalasr** [comment]
Thank you @Deuchnord.

**Comment by chindit** [comment]
Thanks @chalasr and @Deuchnord for your work on this 👍

**Pull request merged**